src/_ert/events.py

```diff
@@ -0,0 +1,211 @@
+import sys
+from datetime import datetime
+from typing import Any, Dict, Literal, Union
+
+if sys.version_info < (3, 9):
+    from typing_extensions import Annotated
+else:
+    from typing import Annotated
+
+from pydantic import BaseModel, Field, TypeAdapter
+
+_FORWARD_MODEL_START = Literal["com.equinor.ert.forward_model_job.start"]
```

Comment:
Should we change the literals to `com.equinor.ert.forward_model_step.*`?

Reply:
changed it to "forward_model_step.start"